### PR TITLE
Update polkadot host

### LINKED ISSUE #277: Update the Polkadot Host page

### PR DESCRIPTION
**Pull request by lsaether**
closes #277 

Tries to be a clearer re-write. The old version was rambly and repeated itself. Also updates to a newer image of the Polkadot runtime / Polkadot host.

### CODE REVIEW

File: docs/learn-polkadot-host.md [hunk 1]
```diff
@@ -4,26 +4,41 @@ title: Polkadot Host (PH)
...
+[specification](https://github.com/w3f/polkadot-spec/blob/master/runtime-environment-spec/polkadot_re_spec.pdf). 
+
+## Components of the Polkadot host
+
+- Networking components such as Libp2p that facilitates network interactions.
 - State storage and the storage trie along with the database layer.
-- Consensus engine.
+- Consensus engine for GRANDPA and BABE (some logic exists on the runtime).
 - Wasm interpreter and virtual machine.
+- Low level primitives for a blockchain, such a cryptographic primitives like hash
```
Comment (Swader) [inline docs/learn-polkadot-host.md]:
```suggestion
- Low level primitives for a blockchain, such as cryptographic primitives like hash
```

File: docs/learn-polkadot-host.md [hunk 2]
```diff
@@ -4,26 +4,41 @@ title: Polkadot Host (PH)
...
+of the Polkadot host while treating the Polkadot runtime as a black box. For more
+details of the interactions between the host and the runtime, please see the
+[specification](https://github.com/w3f/polkadot-spec/blob/master/runtime-environment-spec/polkadot_re_spec.pdf). 
+
+## Components of the Polkadot host
+
+- Networking components such as Libp2p that facilitates network interactions.
 - State storage and the storage trie along with the database layer.
-- Consensus engine.
+- Consensus engine for GRANDPA and BABE (some logic exists on the runtime).
```
Comment (Swader) [inline docs/learn-polkadot-host.md]: I'd clarify the "some logic..." part.

**Review by Swader — CHANGES_REQUESTED** [review]
Nitpicks. Also looks like yarn.lock changes snuck in there?

File: docs/learn-polkadot-host.md (hunk 2 above)
Comment (lsaether) [reply docs/learn-polkadot-host.md]: I'll opt to remove it, going into details here would be out of place.

**Review by Swader — CHANGES_REQUESTED** [review]
Looks good to me only that yarn lock is still in there. Rebasing might help?

**Comment by lsaether** [comment]
It's a more up to date `yarn.lock`. Anyway, it's been reset.

**Pull request merged**